scipy/stats/_distribution_infrastructure.py

```diff
@@ -1212,6 +1212,25 @@ def _logexpxmexpy(x, y):
     return res
 
 
+def _guess_bracket(xmin, xmax):
+    a = np.asarray(-np.ones_like(xmin))
```

Comment:
Can `np.ones_like` not be an array?

Reply:
Huh. Something happened during testing to make me add that. But now when I test it, it does seem that `ones_like` always returns an array. Let's find out.

Reply:
Well I guess that answers our immediate question, but it does leave another.

Reply:
Oh, I remember. It's the negation. Because, you know, obviously a 0-d array shouldn't survive a negation.
LMK if you'd prefer it like it was or if you want this to use `np.full_like` with argument `-1`.

Reply:
`full_like` sounds good